# Absolute encoder counts
int32 abs_encoder_counts
# Motor counts
int32 motor_counts
float64 steering_position
# Status of the absolute encoder
EpcEncoderStatus encoder_status
# Status of the motor drive
robotnik_msgs/MotorStatus motor_status


